experiments/pipeline.jl

```diff
@@ -4,40 +4,12 @@ using Pkg
 experiment_id = ARGS[1]
 Pkg.activate("experiments/$experiment_id")
 Pkg.instantiate()
-using CalibrateAtmos
+import CalibrateAtmos
```

Comment:
Just curious why we need import here, since we're importing in the included files.

Reply:
I don't think we need import here, thank you for catching that!